src/backend/mysql/query.rs

```diff
@@ -58,4 +58,8 @@ impl QueryBuilder for MysqlQueryBuilder {
     ) {
         // MySQL doesn't support declaring materialization in SQL for with query.
     }
+
+    fn insert_default_keyword(&self) -> &str {
+        "VALUES (DEFAULT)"
```

Comment:
it does seem like it's only for inserting one row

Reply:
It will insert only one row with default values